libs/sdk-backend-spi/src/deprecated/dateFilterConfigs/index.ts

```diff
@@ -0,0 +1,177 @@
+// (C) 2007-2022 GoodData Corporation
+/* eslint-disable @typescript-eslint/no-empty-interface, import/namespace */
+import * as m from "@gooddata/sdk-model";
+import {
+    DateFilterOptionAllTimeType,
```

Comment:
Nitpick: can we take this from the `m.` namespace?

Reply:
These are renamed typings and namespacing is not necessary. Should I change it?

Reply:
Oh, ok never mind :)